src/huggingface_hub/hf_api.py

```diff
@@ -5191,6 +5193,9 @@ def get_repo_discussions(
         self,
         repo_id: str,
         *,
+        author: Optional[str] = None,
+        discussion_type: Optional[str] = None,
+        discussion_status: Optional[str] = None,
```

Comment:
I'd add type annotation here to help autocompletion tools

```suggestion
        discussion_type: Optional[Literal[["all", "discussion", "pull_request"]] = None,
        discussion_status: Optional[Literal["all", "open", "closed"]] = None,
```